### PR TITLE
test: move remaining tests to jest

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: test/chromium/tracing.jest.js [hunk 1]
```diff
@@ -16,23 +16,18 @@
...
-      fs.unlinkSync(state.outputFile);
-      state.outputFile = null;
-    }
-  });
+registerFixture('outputFile', async ({parallelIndex}, test) => {
+  const outputFile = path.join(OUTPUT_DIR, `trace-${parallelIndex}.json`);
+  await test(outputFile);
+  if (fs.existsSync(outputFile)) {
+    fs.unlinkSync(outputFile);
+    outputFile = null;
```
Comment (dgozman) [inline test/chromium/tracing.jest.js:26]: Assigning to a const variable?

File: test/chromium/tracing.jest.js [hunk 2]
```diff
@@ -16,23 +16,18 @@
...
-  afterEach(async function(state) {
-    await state.browser.close();
-    state.browser = null;
-    state.page = null;
-    if (fs.existsSync(state.outputFile)) {
-      fs.unlinkSync(state.outputFile);
-      state.outputFile = null;
-    }
-  });
+registerFixture('outputFile', async ({parallelIndex}, test) => {
```
Comment (JoelEinbinder) [inline test/chromium/tracing.jest.js:21]: what happens when I register a fixture in a file? Can it be used in otherfiles? Why not just use beforeEach?

File: test/chromium/oopif.jest.js [hunk 3]
```diff
@@ -14,34 +14,47 @@
...
+  try {
+    await test(browser);
+  } finally {
+    await browser.close();
+  }
+});
+
+registerFixture('sppContext', async ({sppBrowser}, test) => {
+  const context = await sppBrowser.newContext();
+  try {
```
Comment (dgozman) [inline test/chromium/oopif.jest.js:33]: I see many fixtures do `try {} finally {}`. Shouldn't this be default behavior in the harness?

**Review by dgozman — APPROVED** [review]

File: test/chromium/tracing.jest.js (hunk 2 above)
Comment (pavelfeldman) [reply test/chromium/tracing.jest.js:21]: beforeEach can't add into the state...

File: test/chromium/tracing.jest.js (hunk 1 above)
Comment (pavelfeldman) [reply test/chromium/tracing.jest.js:26]: Woohoo!

File: test/chromium/oopif.jest.js (hunk 3 above)
Comment (pavelfeldman) [reply test/chromium/oopif.jest.js:33]: It should, but I often times need to know whether the test succeeded or failed and I don't want to return boolean.

**Pull request merged**